backend/config/urls.py

```
@@ -60,7 +60,7 @@
     path("api/schema/", SpectacularAPIView.as_view(), name="schema"),
     path(
         "api/docs/",
-        SpectacularSwaggerView.as_view(url_name="schema"),
+        SpectacularSwaggerSplitView.as_view(url_name="schema"),
```

Ruff (0.15.20)

[error] Undefined name `SpectacularSwaggerSplitView`

(F821)